File: Beecrowd/python/1019.py
# Problema: [1019] Conversão de Tempo
# Dificuldade: Iniciante - Nível 1
# Autor: Maria Angélica Krüger Miranda

tempo_total_segundo = int(input())

horas= tempo_total_segundo//3600
minutos = (tempo_total_segundo % 3600) // 60
segundos = (tempo_total_segundo % 3600) % 60

print(f"{horas}:{minutos}:{segundos}")

File: Beecrowd/c++/1019.cpp
// Problema: [1019] Conversão de Tempo
// Dificuldade: Iniciante - Nível 1
// Autor: Maria Angélica Krüger Miranda

#include<bits/stdc++.h>

using namespace std;

void fastIO(){
    ios::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);
}

using namespace std;

int main (){
    int segundos;
    cin >> segundos;
    int horas, minutos;
    minutos = segundos/60;
    horas = minutos/60;
    cout << horas << ":" << minutos%60 << ":" << segundos%60 << endl;
    return 0;
}